### PR TITLE
Set PEP 757 status to Accepted

### PR DESCRIPTION
**Pull request by vstinner**
----
📚 Documentation preview 📚: https://pep-previews--4140.org.readthedocs.build/

### CODE REVIEW

File: peps/pep-0757.rst [hunk 1]
```diff
@@ -4,11 +4,12 @@ Author: Sergey B Kirpichev <skirpichev@gmail.com>,
         Victor Stinner <vstinner@python.org>
 PEP-Delegate: C API Working Group
```
Comment (hugovk) [inline peps/pep-0757.rst:5]: Has the SC confirmed the WG delegation for this PEP?

https://github.com/python/steering-council/issues/259 says this PEP is special, and in general there is no standing delegation, but I don't see a clear decision regarding this PEP.

cc @Yhg1s

File: peps/pep-0757.rst [hunk 2]
```diff
@@ -4,11 +4,12 @@ Author: Sergey B Kirpichev <skirpichev@gmail.com>,
...
 Discussions-To: https://discuss.python.org/t/63895
-Status: Draft
+Status: Accepted
 Type: Standards Track
 Created: 13-Sep-2024
 Python-Version: 3.14
 Post-History: `14-Sep-2024 <https://discuss.python.org/t/63895>`__
+Resolution: https://discuss.python.org/t/pep-757-c-api-to-import-export-python-integers/63895/73
```
Comment (hugovk) [inline peps/pep-0757.rst:12]: We can now include the date here too:
```suggestion
Resolution: `28-Nov-2024 <https://discuss.python.org/t/pep-757-c-api-to-import-export-python-integers/63895/73>`__
```

Comment (vstinner) [reply peps/pep-0757.rst:12]: Oh, interesting, I will do the change when I will mark the PEP as Final.

File: peps/pep-0757.rst (hunk 1 above)
Comment (vstinner) [reply peps/pep-0757.rst:5]: PEP 757 was created from https://github.com/capi-workgroup/decisions/issues/35 : it's a working document for the C API Working Group to help to take a decision. I created PEP 757 with `PEP-Delegate: C API Working Group`.

Comment (vstinner) [reply peps/pep-0757.rst:5]: There was a misunderstanding on my side about the C API WG ability to handle their own PEPs.

I submitted [PEP 757 to the Steering Council](https://github.com/python/steering-council/issues/264).

**Pull request merged**